x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/three_way_diff/rule_upgrade/field_upgrade_state_info/translations.tsx

```diff
@@ -61,30 +61,30 @@ export const REVIEWED_AND_ACCEPTED = i18n.translate(
 export const SOLVABLE_CONFLICT = i18n.translate(
   'xpack.securitySolution.detectionEngine.rules.upgradeRules.fieldUpgradeState.solvableConflict',
   {
-    defaultMessage: 'Solved conflict',
+    defaultMessage: 'Auto-resolved conflict',
   }
 );
 
 export const SOLVABLE_CONFLICT_DESCRIPTION = i18n.translate(
   'xpack.securitySolution.detectionEngine.rules.upgradeRules.fieldUpgradeState.solvableConflictDescription',
   {
     defaultMessage:
-      'We have suggested an update for this modified field, please review before accepting.',
+      "We combined your changes with changes from the Elastic update. Review the suggested change to ensure it's correct.",
```

Comment:
Minor tweak to align the sentences. (This was my mistake - sorry about that!)

```suggestion
      "We combined your changes with changes from the Elastic update. Review the suggested changes to ensure they're correct.",
```